docs/docs/networks/composable/LAYR-tokenomics.md

```diff
@@ -45,13 +45,13 @@ The total supply of 100 million LAYR tokens are intended to be distributed and r
 
 **Team**: 25% (25,000,000 LAYR) of the total token supply will be distributed to founders and the core Composable team as a reward as well as incentives for their continued actions in the best interest in the Composable ecosystem. The distribution of these tokens will have a six-month cliff with linear vesting of the remaining tokens over the subsequent two and a half years.
 
-**Token purchasers**: ~23.7% (23,710,772 LAYR) of the total token supply will be distributed to token purchasers as a reward for their essential early support and strategic backing of the ecosystem. 20% (4,742,154.34) of these tokens will be unlocked at the token generation event (TGE), with linear vesting of the remaining tokens over the subsequent two years.
+**Token purchasers**: 23.71% of the total token supply (23,710,772 LAYR) will be distributed to early supporters and strategic backers, with 13.36% (13,359,572 LAYR) in the Seed Allocation, where 20% unlocks at TGE, and the rest vests over 24 months, and 10.36% (10,351,000 LAYR) in the Series A Allocation, with a 6-month vesting period followed by linear vesting over 24 months.
```

Comment:
sir for the 4th time lol just say this: 
23.71% of the total token supply (23,710,772 LAYR) will be distributed to early supporters and strategic backers, with 13.36% (13,359,572 LAYR) unlocking 20% at TGE, and the rest vesting over 24 months, and 10.36% (10,351,000 LAYR) with a 6-month vesting period followed by linear vesting over 24 months.

Reply:
23.71% of the total token supply (23,710,772 LAYR) will be distributed to early supporters and strategic backers. 13.36% (13,359,572 LAYR) of this allocation will  have a 20% unlock at TGE, with the rest vesting over 24 months. 10.36% (10,351,000 LAYR)  of this allocation will be locked for 6-month vesting period followed by linear vesting over 24 months.